### PR TITLE
US-108: Subida de la parte 1-front

### PR DESCRIPTION
**Pull request by edduardaq**

### CODE REVIEW

File: src/app/enrollment/enrollment.service.ts
Comment (ebarahonautn) [inline src/app/enrollment/enrollment.service.ts:1]: @edduardaq where do you capture any error the api can return?

File: src/environments/environment.ts
Comment (ebarahonautn) [inline src/environments/environment.ts:1]: this is the correct endpoint for verification code: **/user/verification**

**Review by ebarahonautn — CHANGES_REQUESTED** [review]
@edduardaq delete the folder **/.angular** before creating pushing your commit.

**Review by ebarahonautn — CHANGES_REQUESTED** [review]
@edduardaq please review the comments added.

**Review by ebarahonautn — CHANGES_REQUESTED** [review]
@edduardaq solve the conflicts in the PR

**Review by LilMat69 — CHANGES_REQUESTED** [review]
Can you please attach the corresponding evidence to this US, in order to verify that the task has the corresponding screen with the requirements.

**Review by LilMat69 — APPROVED** [review]
All the images seems ok, complies with what was requested.

**Review by ebarahonautn — APPROVED** [review]

**Pull request merged**